docs/shell.md

```diff
@@ -158,15 +155,15 @@ If you wish to run a single InSpec command and fetch its results, you
 may use the `-c` flag. This is similar to using `bash -c`.
 
 ```bash
-$ inspec shell -c 'describe file("/Users/ksubramanian") do it { should exist } end'}
+$ inspec shell -c 'describe file("/Users/myuser") do it { should exist } end'}
 Target:  local://
 
-  ✔  File /Users/ksubramanian should exist
+  ✔  File /Users/myuser should exist
 
 Summary: 1 successful, 0 failures, 0 skipped
 ```
 
 ```bash
-$ inspec shell --format json -c 'describe file("/Users/ksubramanian") do it { should exist } end'
-{"version":"0.30.0","profiles":{"":{"supports":[],"controls":{"(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)":{"title":null,"desc":null,"impact":0.5,"refs":[],"tags":{},"code":"          rule = rule_class.new(id, profile_id, {}) do\n            res = describe(*args, &block)\n          end\n","source_location":{"ref":"/Users/ksubramanian/repo/chef/inspec/lib/inspec/profile_context.rb","line":184},"results":[{"status":"passed","code_desc":"File /Users/ksubramanian should exist","run_time":0.000747,"start_time":"2016-08-16 11:41:40 -0400"}]}},"groups":{"in_memory.rb":{"title":null,"controls":["(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)"]}},"attributes":[]}},"other_checks":[],"summary":{"duration":0.001078,"example_count":1,"failure_count":0,"skip_count":0}}}
+$ inspec shell --format json -c 'describe file("/Users/myuser) do it { should exist } end'
+{"version":"0.30.0","profiles":{"":{"supports":[],"controls":{"(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)":{"title":null,"desc":null,"impact":0.5,"refs":[],"tags":{},"code":"          rule = rule_class.new(id, profile_id, {}) do\n            res = describe(*args, &block)\n          end\n","source_location":{"ref":"/Users/myuser/repo/chef/inspec/lib/inspec/profile_context.rb","line":184},"results":[{"status":"passed","code_desc":"File /Users/myuser should exist","run_time":0.000747,"start_time":"2016-08-16 11:41:40 -0400"}]}},"groups":{"in_memory.rb":{"title":null,"controls":["(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)"]}},"attributes":[]}},"other_checks":[],"summary":{"duration":0.001078,"example_count":1,"failure_count":0,"skip_count":0}}}
```

Comment:
Would this be more meaningful for the example?

```
{
  "version": "0.30.0",
  "profiles": {
    "": {
      "supports": [],
      "controls": {
        "(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)": {
          "title": null,
          "desc": null,
          "impact": 0.5,
          "refs": [],
          "tags": {},
          "code": "          rule = rule_class.new(id, profile_id, {}) do\n            res = describe(*args, &block)\n          end\n",
          "source_location": {
            "ref": "/Users/myuser/repo/chef/inspec/lib/inspec/profile_context.rb",
            "line": 184
          },
          "results": [{
            "status": "passed",
            "code_desc": "File /Users/myuser should exist",
            "run_time": 0.000747,
            "start_time": "2016-08-16 11:41:40 -0400"
          }]
        }
      },
      "groups": {
        "in_memory.rb": {
          "title": null,
          "controls": ["(generated from in_memory.rb:1 5aab65c33fb1f133d9244017958eef64)"]
        }
      },
      "attributes": []
    }
  },
  "other_checks": [],
  "summary": {
    "duration": 0.001078,
    "example_count": 1,
    "failure_count": 0,
    "skip_count": 0
  }
}
}
```

Reply:
I think so! Thanks for all of your thoughtful feedback!